feature-libs/cart/base/core/store/selectors/multi-cart.selector.spec.ts

```diff
@@ -194,21 +195,58 @@ describe('Multi Cart selectors', () => {
   });
 
   describe('getCartIsStableSelectorFactory', () => {
-    it('should return cart stability flag', () => {
-      let result;
+    it('should return true when cart is stable when there is no active cart', (done) => {
```

Comment:
when I say there is no active cart, it means active is currently set to empty string AND if you look at the cart process state, it does not have the processCount set as active cart is empty